domains/shivampanchal.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "shiv1819",
+        "email": "shivampanchal11q@gmai.com"
+    },
+    "record": {
+        "TXT": "c63bccf2a3f395ca9e30331302464e"
```

Comment:
TXT doesn't do anything. Was this by any chance a Github Pages TXT verification? Because if so thats not how you do it.

Reply:
yes it was  a Github Pages TXT verification